app.vup.Chat.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: app.vup.Chat
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: vup_chat
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--socket=pulseaudio"
+  - "--share=network"
+  - "--talk-name=org.kde.StatusNotifierWatcher"
+  - "--talk-name=org.freedesktop.Notifications"
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - name: libnotify
+    buildsystem: meson
+    config-opts:
+      - -Dman=false
+      - -Dgtk_doc=false
+      - -Ddocbook_docs=disabled
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /lib/debug
+      - /lib/girepository-1.0
+      - /share/gtk-doc
+      - /share/gir-1.0
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/libnotify.git
+        tag: 0.8.3
+
+  - name: vup-chat
+    buildsystem: simple
+    build-commands:
+      - install -d /app/vup-chat
+      - cp -r build/* /app/vup-chat
+      - chmod +x /app/vup-chat/vup_chat
+      - mkdir -p /app/bin
+      - ln -s /app/vup-chat/vup_chat /app/bin/vup_chat
+      - install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/app.vup.Chat.png
+      - install -Dm644 app.vup.Chat.desktop -t /app/share/applications/
+      - install -Dm644 app.vup.Chat.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/vup-app/vup-chat/main/static/icon.svg
+        sha256: b80bbe5893595b1f47497de3400472492878550ac3abc504c40dd24468ae9c3d
+      - type: file
+        url: https://raw.githubusercontent.com/vup-app/vup-chat/main/linux/metadata/app.vup.Chat.metainfo.xml
+        sha256: 895308d37fa095a5f604dc5343f0c0b67c4a8ccac27a379d9f184f10fe23cf61
+      - type: file
+        url: https://raw.githubusercontent.com/vup-app/vup-chat/main/linux/metadata/app.vup.Chat.desktop
+        sha256: 8a87ae28fbd38321f7427692a10c405a61c32f72d584aa682f7ceeb488c276b9
+      - type: archive
+        dest: build
+        only-arches:
+          - x86_64
+        url: https://github.com/vup-app/vup-chat/releases/download/v0.5.8/vup-chat-linux.zip
```

Comment:
Why not build from source?

Reply:
Last time I tried to package a flutter app, building from source wasn't an option because it inherently requires an internet connection and flatpak builds aren't conducive to this because they try to be as reproducible as possible. This could have changed though? Do let me know.

Reply:
Ok, yea building flutter toolchain does not work in a Flatpak environment

Reply:
I would like to make this multiarch, but that isn't something I can really test right now (my gear is x86 only unfortunately) so I'm kicking that can down the road :p